### DISCUSSION TITLE
Ability to add a prefix to Quasar CSS classes

### DISCUSSION THREAD

**Discussion by gustavotoyota** [quasarframework/quasar#16692] [Ideas / Proposals]
I want to work with Tailwind, but I don't want to have to add a prefix to Tailwind classes.

The opposite should be possible: use Tailwind without prefixes, and add a prefix to Quasar classes.

No more conflict between Quasar and Tailwind.

**Comment by ScreamZ** [comment]
Interested in this too

**Comment by danielunderpinned** [comment]
+1, why is it so hard to use tailwind with quasar !!!!!

**Reply by ScreamZ** [reply]
Because it has been designed with hardcoded classes and this is a huge work to rework it.

You can try to make a custom tailwind plugin and redefine only classes that conflict, I created a "hide" class instead of "hidden"
